caddy/caddy_test.go

```diff
@@ -1500,3 +1500,269 @@ func TestLog(t *testing.T) {
 		"",
 	)
 }
+
+// TestSymlinkNeighboringWorkerScript tests executing a worker script from within a symlinked directory
+// Scenario: neighboring worker script
+//
+//	Given frankenphp located in the test folder
+//	When I execute `frankenphp php-server --listen localhost:8080 -w index.php` from `public`
+//	Then I expect to see the worker script executed successfully
+func TestSymlinkNeighboringWorkerScript(t *testing.T) {
```

Comment:
Couldn't you use [subtests](https://go.dev/blog/subtests) instead a separated tests?

Reply:
Oh and Happy New Year!